client/consensus/grandpa/blockOrigin.go

```diff
@@ -0,0 +1,70 @@
+package grandpa
```

Comment:
This file nomenclature pattern is different from others, I would suggest renaming to `block_origin.go`

Reply:
Actually, with the changes I had made this type is no longer used, so I am removing it